packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -271,8 +271,13 @@ export function checkInstalled(requested: string) {
 	);
 }
 
-export const loadPackage = async (modulePath: string, pkg: string): Promise<any> =>
-	import(pathToFileURL(path.join(modulePath, "node_modules", pkg, "dist", "index.js")).href);
+export const loadPackage = async (modulePath: string, pkg: string): Promise<any> => {
+	const pkgPath = path.join(modulePath, "node_modules", pkg);
+	const pkgJson: { main: string } = JSON.parse(
```

Comment:
This should probably use the exports field if it exists instead of assuming the main field is correct. If we use the main field, we may use different code than node itself loads.